### PR TITLE
GPT4 analysis on Paul Graham essay

### PR DESCRIPTION
**Pull request by Disiok**

### CODE REVIEW

File: gpt_index/langchain_helpers/chain_wrapper.py [hunk 1]
```diff
@@ -41,6 +45,19 @@ def _get_llm_metadata(llm: BaseLanguageModel) -> LLMMetadata:
             max_input_size=llm.modelname_to_contextsize(llm.model_name),
             num_output=llm.max_tokens,
         )
+    elif isinstance(llm, ChatOpenAI):
+        if llm.model_name == 'gpt-4':
```
Comment (jerryjliu) [inline gpt_index/langchain_helpers/chain_wrapper.py:49]: is this integrated into langchain now?

**Review by jerryjliu — COMMENTED** [review]
@Disiok do you plan to merge this nb?

**Review by jerryjliu — APPROVED** [review]
if so i'll approve to unblock

File: gpt_index/langchain_helpers/chain_wrapper.py (hunk 1 above)
Comment (Disiok) [reply gpt_index/langchain_helpers/chain_wrapper.py:49]: Let me check latest, and revise this PR.

Comment (Disiok) [reply gpt_index/langchain_helpers/chain_wrapper.py:49]: Nope, not available yet as of 0.0.115.

Comment (jerryjliu) [reply gpt_index/langchain_helpers/chain_wrapper.py]: huh

**Review by jerryjliu — APPROVED** [review]

**Pull request merged**